bot/src/main.rs

```diff
@@ -2442,29 +2442,41 @@ fn generate_csv(epoch: Epoch, config: &Config) -> BoxResult<()> {
 }
 
 /// Given a validator's current commission and history of commission changes, returns the validator's commission at the end of `epoch`
+/// Only works if the commission change history includes all changes for `epoch` and `epoch + 1`.
 fn calculate_commission_at_end_of_epoch(
     epoch: u64,
     current_commission: u8,
```

Comment:
Where is the current_commission coming from? On chain or somewhere else?

Reply:
The one place this function is called, it is retrieved from the get_vote_accounts RPC call.